### PR TITLE
Move release process docs to handbook

### PR DESCRIPTION
**Pull request by jasonblais**
Moving legacy handbook to handbook.mattermost.com is tracked as part of https://docs.google.com/spreadsheets/d/1Yz93lett6y-vPqyxqBFZhL_43xBQ2eYRUwcVyEat-lo/edit#gid=0

Accompanying docs PR: https://github.com/mattermost/docs/pull/3542

### CODE REVIEW

File: operations/research-and-development/product/release-process/release-scorecard-definitions.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+<li>Issue Type = Bug</li>
+<li>Label = P3</li>
+<li>Created Date = 17th of the previous month</li>
+<li>Release Date = 16th of the current month</li>
+</ol>
+<p>Copy a list of Jira tickets with the above information and paste them into a spreadsheet. Calculate the average by using a formula “=Release Day-Created”.</p></td>
+</tr>
+<tr class="odd">
+<td>Number of P1 bugs reported in production</td>
+<td>Bug severity guidelines are in progress, after which a Jira query will be built</td>
```
Comment (jasonblais) [inline operations/research-and-development/product/release-process/release-scorecard-definitions.md]: Would we be ready to add the Jira query you use?

**Comment by amyblais** [comment]
~~Amy: add release checklist step to create a ticket for Platfrom to add telemetry.~~

File: operations/research-and-development/product/release-process/release-scorecard-definitions.md (hunk 1 above)
Comment (amyblais) [reply operations/research-and-development/product/release-process/release-scorecard-definitions.md]: Added a guideline

**Review by amyblais — DISMISSED** [review]

File: operations/research-and-development/product/release-process/release-scorecard-definitions.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+<li>Issue Type = Bug</li>
+<li>Label = P3</li>
+<li>Created Date = 17th of the previous month</li>
+<li>Release Date = 16th of the current month</li>
+</ol>
+<p>Copy a list of Jira tickets with the above information and paste them into a spreadsheet. Calculate the average by using a formula “=Release Day-Created”.</p></td>
+</tr>
+<tr class="odd">
+<td>Number of P1 bugs reported in production</td>
+<td>Count of major issues reported in documentation or marketing by customers, CSMs, Support or Product. E.g. dot releases from customer reports, undocumented breaking changes.</td>
```
Comment (jasonblais) [inline operations/research-and-development/product/release-process/release-scorecard-definitions.md]: @amyblais Did you intend to include issues in product as well?
```
Count of major issues reported in product, documentation or marketing by customers, CSMs, Support or Product. E.g. dot releases from customer reports, undocumented breaking changes.
```

**Pull request merged**